packages/react-bindings/README.md

```diff
@@ -29,13 +31,70 @@ yarn add @stardust-ui/react-bindings
 
 # Hooks
 
+## `useAccesibility()`
+
+A React hook that provides bindings for accessibility behaviors.
+
+#### Usage
+
+The example below assumes a component called `<Input>` will be used this way:
```

Comment:
```suggestion
The example below assumes a component called `<Image>` will be used this way:
```

Isn't the example for `Image`? Also, not sure we are providing some example of how it is used, it's rather how it is defined

Reply:
> Also, not sure we are providing some example of how it is used, it's rather how it is defined

Sorry, don't understand fully, how to use this `Image` component?